CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ _None_
 
 ### New Features
 
-_None_
+- Introduce new `openai_generate` action to get responses to a prompt/question from OpenAI API. [#621]
```

Comment:
FYI: I forgot to update the CHANGELOG before merging this PR, but I've fixed it directly in `trunk` afterwards, via 1a58ba02